src/portal.cpp

```diff
@@ -820,7 +820,20 @@ DVector2 P_GetOffsetPosition(double x, double y, double dx, double dy)
 	return dest;
 }
 
+static void GetOffsetPosition(double x, double y, double dx, double dy, DVector2 *result)
+{
+	*result = P_GetOffsetPosition(x, y, dx, dy);
+}
 
+DEFINE_ACTION_FUNCTION_NATIVE(DObject, GetOffsetPosition, GetOffsetPosition)
+{
+	PARAM_PROLOGUE;
+	PARAM_FLOAT(x);
+	PARAM_FLOAT(y);
+	PARAM_FLOAT(dx);
+	PARAM_FLOAT(dy);
+	ACTION_RETURN_VEC2(P_GetOffsetPosition(x, y, dx, dy));
+}
 //============================================================================
```

Comment:
I'm sorry, but is it too hard to add empty line here?

Reply:
That was an oversight. Fixed.